src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests.cs

```diff
@@ -2819,8 +2821,8 @@ interface Bar
                 Punctuation.Semicolon,
                 Keyword("const"),
                 Keyword("int"),
-                Constant("goo3"),
                 Static("goo3"),
+                Constant("goo3"),
```

Comment:
I don't understand why this had to change, but this made the test happy. I've _fairly_ sure its not something I've done, since the `goo3` constant appears above the delegates I added ¯\\\_(ツ)_/¯

Reply:
i would just not change this test.

Reply:
Well, if the test was passing before your change, then it must be _something_ you did. :smile: Given this would be returning multiple classifications for the same span (it's both a constant and a static?) then I wonder if it's not stable, and so when you added other stuff it caused whatever underlying data structure to change how it's structured? Like a SortedList rebalanced in some odd way somehow?

Reply:
we put the items in the list and we sort.  but it's highly likely it's not a stable sort.

Reply:
I changed this to use a stable sort, which resulted in a couple more changes. I suspect adding ClassificationType to the sort would have had the same effect.

@CyrusNajmabadi @jasonmalinowski before I merge, can you confirm that this is an okay change? It didn't have a noticeable difference in test run time on my machine.